### PR TITLE
Add API key test

### PR DESCRIPTION
**Pull request by daniel8702**

### CODE REVIEW

File: src/Creator/ApiKeysTestCreator.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+namespace BitBag\SyliusMolliePlugin\Creator;
+
+use BitBag\SyliusMolliePlugin\Client\MollieApiClient;
+use BitBag\SyliusMolliePlugin\DTO\ApiKeyTest;
+use Mollie\Api\Resources\MethodCollection;
+
+final class ApiKeysTestCreator implements ApiKeysTestCreatorInterface
+{
+    /** @var MollieApiClient */
```
Comment (patrick477) [inline src/Creator/ApiKeysTestCreator.php:20]: interface?

File: src/Resources/views/Admin/PaymentMethod/testApiKeys.html.twig [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class="ui message">
+{% for test in tests if tests is not empty %}
```
Comment (patrick477) [inline src/Resources/views/Admin/PaymentMethod/testApiKeys.html.twig:2]: add tab ->

File: src/Resources/views/Admin/PaymentMethod/_testApiKey.html.twig [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    {% endif %}
+{% endif %}
+    {% if test.key == false %}
+        {{ 'bitbag_sylius_mollie_plugin.ui.inser_you_key_first'|trans }}
+    {% endif %}
+    {{ 'bitbag_sylius_mollie_plugin.ui.available_methods'|trans }}:
+    {% for method in test.methods %}
+        {{ method.description }},
+    {% endfor %}
+</div>
```
Comment (patrick477) [inline src/Resources/views/Admin/PaymentMethod/_testApiKey.html.twig:24]: :)

File: src/Creator/ApiKeysTestCreator.php (hunk 1 above)
Comment (daniel8702) [reply src/Creator/ApiKeysTestCreator.php:20]: no interface in MollieApiClient :)

Comment (patrick477) [reply src/Creator/ApiKeysTestCreator.php:20]: it would be nice to add an interface :)

**Review by patrick477 — APPROVED** [review]

File: src/Creator/ApiKeysTestCreator.php (hunk 1 above)
Comment (daniel8702) [reply src/Creator/ApiKeysTestCreator.php:20]: Yes true but base MollieApiClient from mollie packages don't have an interface so if I create myself can cause problems

**Pull request merged**